content/guide/development-workflow/using-packages.md

```diff
@@ -0,0 +1,104 @@
+<!-- TODO: fix links -->
+
+Also, make sure to look through the [NativeScript core modules](https://docs.nativescript.org/understanding-packages.html#nativescript-core), which ship as a dependency of every NativeScript app. There’s a chance that the functionality you need is built in. 
+
+#### Installing Plugins
+
+Once you’ve found the plugin you need, install the plugin into your app using the `ns plugin add` command.
+
+```cli
+ns plugin add <plugin-name>
+```
+
+For example, the following command installs the [NativeScript camera plugin](https://docs.nativescript.org/plugins/camera.html).
+
+```cli
+ns plugin add @nativescript/camera
+```
+
+Instead of using `plugin add`, you can use your package manager as well (npm, yarn, pnpm...):
+
+```cli
+npm install --save @nativescript/camera
+```
+
+The installation of a NativeScript plugin mimics the installation of an npm package. The NativeScript CLI downloads the plugin from npm and adds the plugin to the `node_modules` folder in the root of your project. During this process, the NativeScript CLI adds the plugin to your project’s root `package.json` file and also resolves the plugin’s dependencies (if any).
+
+#### Installing Plugins as Developer Dependencies
+
+As shown above the command `ns plugin add @nativescript/camera` is actually doing `npm i @nativescript/camera --save` behind the scenes. If you need to install a **developer dependency** in your project (e.g., like **@nativescript/types** or **@nativescript/webpack**) then you will need to explicitly save it as a **devDependency**. To achieve that, use the `npm install` command with `--save-dev` flag. For example:
+
+```cli
+npm i @nativescript/types --save-dev
+```
+
+::: tip Note
+The difference between dependencies and developer dependencies is that **dependencies** are required to run, while **devDependencies** are needed only during development. Example for dependency is the **@nativescript/camera** plugin which is required at runtime so you could use the hardware camera. On the other hand, the **@nativescript/types** is a developer dependency required only for intelliSense during the development process. The `devDependencies` should not be installed as `dependencies` to avoid large output build files (large application size). Example `package.json` file using both `dependencies` and `devDependencies` can be found [here](https://github.com/NativeScript/nativescript-sdk-examples-js/blob/master/package.json#L31-L44).
+:::
+
+#### Importing and Using Plugins
+
+Once the plugin you need is installed, you can start using it in your project. Note that each plugin might have its configuration that needs to be satisfied so always check carefully the plugin's documentation and the README file. The below code snippet demonstrated the basic usage of **@nativescript/camera** plugin.
```

Comment:
```suggestion
Once the required plugin is installed, it can be utilized in the project.  It's crucial to carefully review the plugin's documentation and README file as each plugin may have specific configuration requirements.  The following code snippet demonstrates the basic usage of the @nativescript/camera plugin:
```